packages/manager/modules/pci/src/projects/new/routing.js

```diff
@@ -99,9 +100,10 @@ export default /* @ngInject */ ($stateProvider) => {
           : '',
 
       cart: /* @ngInject */ ($transition$, me, pciProjectNew) => {
-        return !get($transition$.params(), 'cartId')
-          ? // just create cart - location will be reloaded to fetch the whole cart
-            pciProjectNew.createOrderCart(me.ovhSubsidiary)
+        const hasCartId = has($transition$.params(), 'cartId');
```

Comment:
**nitpick**
To avoid import of `has` from lodash
```suggestion
        const hasCartId = $transition$.params().hasOwnProperty('cartId');
```

Reply:
Thanks for suggestion, i prefer `has` because is shortest and readable.